pages/library_implementation.md

```diff
@@ -0,0 +1,125 @@
+
+A Planet resource is any material that has been uploaded into the digital library. There are many types of resources. PLANET can accommodate PDFs (books), MP3s (audio), MP4s (videos), HTML5 applications (interactive content), and just about any other format. Resources can be viewed in the library . The make of the the library can vary from Communities to Community and from Nation to Nation. Resources may also stored in a variety of languages, including (but not limited to) English, Kiswahili, French, Spanish, Arabic, and Somali.
+
+## What is a Collection?
+
+A collection is a group of resources that are grouped by attribute such as topic/subject, grade level, intended use, etc. PLANET currently follows a the Library of Congress categorization system. Collections exist as the main category. Within a collection resources can be further grouped into sub-collections to further define and categorize the collection. Collections are alphabetized by topic.
+
+# Library Navigation
+
+## How do I upload a Resource?
+
+1.  Log in as a **Learner**, **Leader**, or **Manager**.
+
+2.  From the main **_Dashboard_** click on the **_Library_** menu item at the top of the page.
+
+3.  After clicking on the Library heading, choose **_“Add New Resource.”_** You should be taken to the following empty screen:
+
+![](uploads/images/NewResource.png)
+
+1.  The **Title** of the resource goes here. Capitalize all principal words and make sure everything is spelled correctly. If there are any dashes in the title, leave a space on either side (i.e., “Addition - Subtraction” NOT “Addition-Subtraction”). Do not start the title with numbers or special characters (!@#$%^&*). If the title is already taken (for example, a version in another language has already been uploaded) include the language of the new resource in parentheses. Examples are shown below.
+
+          a. A Tale of Two Cities
+          b. A Tale of Two Cities (Spanish)
+          
+2. Type the name of Author(s)/Editor. Authors should be listed as follows: “FIRST NAME, LAST NAME.” Illustrators, editors, translators, etc., should be identified in this section with their title in parentheses.
+
+          a. John Doe
+          b. Jane Doe (Translator)
+          
+3. **_Publisher/Attribution_** is mandatory for copyright purposes. Please do not leave this field blank. Type the name of the organization or publisher (and any websites associated with the resource, if applicable). Attribution allows OLE to use the source under Creative Commons or copyright and protects against copyright laws violations.
+
+4. If available, type the **_Year_** in which the resource was published.
+
+5. **_Link to License_** is also mandatory for copyright purposes. Locate the copyright information for the resource. If it has a Creative Commons license, locate the correct license from the Creative Commons website and paste the URL in the "Link to License" field.  Type “Public Domain” if the resource is in the public domain.
```

Comment:
Can we add the link about licenses? Is [this](https://creativecommons.org/licenses/#licenses) the Creative Commons page you are referring to?

Reply:
+1

Reply:
I am not sure as this text was not written by me. However, I don't believe this is a single link to one license. Rather, it indicates that the link to the author's license declaration should go here. I could be wrong in which case we could provide a link info on CC: licenses such as [Wikipedia details here](https://en.wikipedia.org/wiki/Creative_Commons_license) or the [CC homepage here](https://creativecommons.org/) or [the CC license overview page here](https://creativecommons.org/licenses/)